site/content/docs/5.2/components/placeholders.md

```diff
@@ -126,7 +126,7 @@ The size of `.placeholder`s are based on the typographic style of the parent ele
 
 ### Animation
 
-Animate placeholders with `.placeholder-glow` or `.placeholder-wave` to better convey the perception of something being _actively_ loaded.
+Animate placeholders with `.placeholder-glow` or `.placeholder-wave` to better convey the perception of something being *actively* loaded.
```

Comment:
Just a question here. Based on the rest of the diff, I suppose that a rule regarding emphasis is set (like [MD049](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md049---emphasis-style-should-be-consistent)). But the other modifications are `*word*` → `_word_`. Do you know why is it in the other way here?

Reply:
Yeah, because the default is consistent, see the description.

I don't mind enforcing it, but this way we have the minimum changes.

Reply:
I don't have any issues with the consistency but my point is that in the modifications of this PR you have:
* 7 times `*word*` -> `_word_`
* 1 time (here) `_word_` -> `*word*` (the opposite)

The latter seems weird to me since it breaks the consistency; everything should be `*word*` at the end shouldn't it?